src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
Consider using a more appropriate data structure for storing fruits. Since you might need to perform operations like checking if a fruit is already in the storage or updating the quantity of a fruit, using a Map<Fruit, Integer> could be more efficient.